src/clr-addons/quick-list/add-option/add-option.component.scss

```diff
@@ -0,0 +1,11 @@
+.btn-thrash {
+  margin-bottom: 0;
+  margin-top: 0.75rem;
+}
+.quick-list-input {
+  flex: 1;
+}
+
+:host(.quick-list-option) {
```

Comment:
host selector in css shouldnt be used anymore - use the host property your @Component annotation to apply certain classes to your host element

Reply:
or @HostBinding annotation.
you should then be able to get rid of the :host()

Reply:
is this a proper solutions now ? 
@HostBinding('style.display') display = 'flex';

Reply:
fixed